src/webui/www/private/scripts/search.js

```diff
@@ -187,7 +187,7 @@ window.qBittorrent.Search ??= (() => {
         // reinitialize tabs
         $("searchTabs").getElements("li").removeEvents("click");
         $("searchTabs").getElements("li").addEvent("click", function(e) {
-            $("startSearchButton").textContent = "QBT_TR(Search)QBT_TR[CONTEXT=SearchEngineWidget]";
+            document.getElementById("startSearchButton").lastChild.nodeValue = "QBT_TR(Search)QBT_TR[CONTEXT=SearchEngineWidget]";
```

Comment:
Could you please explain the change? are those necessary? (about `lastChild.nodeValue` part)

Reply:
> Setting textContent on a node removes all of the node's children and replaces them with a single text node with the given string value.

I could not longer keep textContent because now there's an icon within the button and it would get constantly erased during RSS search.

In this case lastChild refers to text node contained in the button and that's the only part what we want to change.

> If the node is a CDATA section, a comment, a processing instruction, or a text node, textContent returns, or sets, the text inside the node, i.e., the Node.nodeValue."

So technically I could have set it to lastChild.textContent but it's just the same thing.

Reply:
>In this case lastChild refers to text node contained in the button and that's the only part we want to change.

Understood.

>So technically I could have set it to lastChild.textContent but it's just the same thing if I'm not wrong.

Yes, but I would prefer that. It is clearer to see what kind of node it is operating on.